### ISSUE TITLE
<base> tag breaks svg id's

### ISSUE THREAD

**Issue by thomas-darling** [aurelia/router#382]
This is more of a FYI than an actual issue, as we did eventually make things work.
However, since this came as quite a surprise to us, I thought it might be a good idea to make you aware of this _potential_ issue, so we don't get any breaking changes in the future.

Basically, be aware that the presence of a `<base>` tag on the page breaks SVG id's, effectively making it impossible to reference any SVG defs, such as filters, gradients, symbols, etc.
In some apps, like ours, this means we simply cannot use a `<base>` tag - instead, we have to set the SystemJS `baseUrl` and use full, absolute paths for things like images, etc.

Here's another issue that describe the problem nicely:
https://github.com/meanjs/mean/issues/1224

I've seen the `<base>` tag mentioned in both the documentation and in several other issues in here, so I thought I'd make you aware of this, so you can make sure any changes made to things like the router or loader don't have a hard dependency on the presence of this tag. It's maybe also something that could be mentioned in the documentation, to save others from the frustration :-)

**Comment by EisenbergEffect** [comment]
Perhaps you would like to submit a PR to the router configuration documentation to add a note about this?

**Comment by thomas-darling** [comment]
It's been almost 8 years since I opened this issue - and I've just now realized that it appears to be resolved 🥳

As mentioned in [this comment](https://issues.chromium.org/issues/40411128#comment23), the SVG spec was actually changed, shortly before I opened this issue, to clarify that fragment-only URLs should in fact _always_ resolve as same-document URLs, even in the presence of a `<base>` element.

Similarly, somewhere along the way, it appears the CSS spec was also [updated](https://drafts.csswg.org/css-values-3/#local-urls) to resolve the issue.

After those spec changes were made, it took a while for browser vendors to fix things on their side - and later, a couple of buggy browser versions accidentally broke it again, adding to the confusion.

The fact seems to be though, that adding a `<base>` element should no longer cause any problems with SVG or CSS, and based on my tests, at the time of writing, this does indeed appear to work as expected in all major browsers.
